docker-compose.yaml

```diff
@@ -174,6 +174,19 @@ services:
       - POSTGRES_USER=${DATABASE_USER}
     volumes:
       - ./temporal/dynamicconfig:/etc/temporal/config/dynamicconfig
+  airbyte-cron:
+    image: airbyte/cron:${VERSION}
+    logging: *default-logging
+    container_name: airbyte-cron
+    restart: unless-stopped
+    environment:
+      - DB=postgresql
```

Comment:
This is not need now but it will be needed in the future for sure. It is better to put it now.